Resources/doc/index.rst

```diff
@@ -220,7 +199,7 @@ You can skip single migrations by explicitly adding them to the ``migration_vers
 
 .. code-block:: terminal
 
-    $ php bin/console doctrine:migrations:version YYYYMMDDHHMMSS --add
+    $ php bin/console doctrine:migrations:version FQCN --add
```

Comment:
Related to the comment above, we may want to use a sample class name to better explain usage.

Reply:
done